resources/template.jade

```diff
@@ -3,7 +3,12 @@ html
 		include head.jade
 	body
 		p
-			em trntxt.uk 
+			em trntxt.uk
 				small (beta)
 		!= content
+
+		if toStation != undefined and fromStation != undefined
```

Comment:
Thats odd. I used an online Jade test thing and `and` was fine. Jade is horrible!